bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -89,31 +88,42 @@ async def nomination_autoreview_group(self, ctx: Context) -> None:
 
     @nomination_autoreview_group.command(name="enable", aliases=("on",))
     @has_any_role(Roles.admins)
+    @commands.max_concurrency(1)
     async def autoreview_enable(self, ctx: Context) -> None:
         """
         Enable automatic posting of reviews.
 
-        This will post reviews up to one day overdue. Older nominations can be
-        manually reviewed with the `tp post_review <user_id>` command.
+        A review will be posted when the current number of active reviews is below the limit
+        and long enough has passed since the last review.
+
+        Users will be considered for review if they have been in the talent pool past a
+        threshold time.
+
+        The next user to review is chosen based on the number of nominations a user has,
+        using the age of the first nomination as a tie-breaker (oldest first).
```

Comment:
I wonder how good this is, since there's a potential starvation here for nominees with only one nomination. What I would do is take both parameters and give each of them a weight to create a total score. It would then review the nominee with the highest score.

Just off the top of my head, we could do something like:
number_of_nominations / number_for_currently_most_nominated  +  age / age_of_currently_oldest

This could be weighted for example to give the number of nominations more importance.

Reply:
> If the talentpool is constantly growing, then so will the age of the oldest nomination in it. If the weighting of the next user picked is dependant on the age and number of nomination, then I don't know how you'd avoid the number of nominations required to be picked would also constantly growing

That's why I normalized the age by the age of the oldest vote. The contribution is capped at 1, and the older the oldest vote, the less of an effect its aging will have on the score.

Meaning, for the formula to give a nomination that isn't the oldest, the gap between the number of nominations needs to be large enough, but it's always capped by the largest number of nominations for any currently nominated user. If the oldest nomination has many nominations it'll be invoked, but if all the old nominations have only a few, and then a newer nomination has many, it has a chance to be invoked. If you multiple the number of nominations in the formula by say 1.5 or 2, it'll be even easier.

Reply:
And, if a nomination is so old it still dominates the score, then maybe it's time we took a look at it?

Reply:
The bottom line is, instead of using the age as a tie breaker, you can use it as a regularization parameter, and strike a balance between popularity and age.

Reply:
Alright, I'm still not convinced that it's really necessary, but i'll try something.

Reply:
If you'd rather keep it simple we can put it aside for now. I can show you later on Discord how the scoring will look like with the current list.